From f085d71492a91a26ca99cb30c1cb7c8b408636eb Mon Sep 17 00:00:00 2001
From: Jerome Brunet <jbrunet@baylibre.com>
Date: Wed, 11 Aug 2021 11:46:03 +0200
Subject: [PATCH] HACK: meson: config: only uart by default

By default, set stdin and stdout to use the uart only.
video and usb keyboard messes with copy/paste in the uart which is
extremely annoying.

If necessary, video and usb can be enabled by reseting the env variables

Signed-off-by: Jerome Brunet <jbrunet@baylibre.com>
---
 include/configs/meson64.h | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/include/configs/meson64.h b/include/configs/meson64.h
index 0d8d320d04d5..0637b87b483a 100644
--- a/include/configs/meson64.h
+++ b/include/configs/meson64.h
@@ -23,18 +23,8 @@
 		230400, 250000, 460800, 500000, 1000000, 2000000, 4000000, \
 		8000000 }
 
-/* For splashscreen */
-#ifdef CONFIG_VIDEO
-#define STDOUT_CFG "vidconsole,serial"
-#else
 #define STDOUT_CFG "serial"
-#endif
-
-#ifdef CONFIG_USB_KEYBOARD
-#define STDIN_CFG "usbkbd,serial"
-#else
 #define STDIN_CFG "serial"
-#endif
 
 #define CONFIG_SYS_SDRAM_BASE		0
 
-- 
2.30.2

